pkg/action/install.go

```diff
@@ -663,6 +663,14 @@ func (c *ChartPathOptions) LocateChart(name string, settings *cli.EnvSettings) (
 		RepositoryConfig: settings.RepositoryConfig,
 		RepositoryCache:  settings.RepositoryCache,
 	}
+
+	if strings.HasPrefix(name, "oci://") {
+		if version == "" {
+			return "", errors.Errorf("--version flag is explicitly required for OCI registries")
```

Comment:
Should not assume CLI

Reply:
resolved in 3e17c1a7efe7396b6c9cda5dbdb1a46a759e4b91